src/ensembl/src/content/app/entity-viewer/gene-view/components/gene-view-sidebar/overview/Overview.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import ExternalLink from 'src/shared/components/external-link/ExternalLink';
+import MainAccordion from './MainAccordion';
+import PublicationsAccordion from './PublicationsAccordion';
+
+import {
+  getEntityViewerSidebarPayload,
+  getEntityViewerSidebarUIState
+} from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarSelectors';
+import { updateEntityUI } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarActions';
+
+import { RootState } from 'src/store';
+import JSONValue from 'src/shared/types/JSON';
+import { EntityViewerSidebarPayload } from 'src/content/app/entity-viewer/state/sidebar/entityViewerSidebarState';
+
+import styles from './Overview.scss';
+
+type Props = {
+  sidebarPayload: EntityViewerSidebarPayload | null;
+  sidebarUIState: { [key: string]: JSONValue } | null;
```

Comment:
not used in the component?

Reply:
It is passed down to `MainAccordion` & `PublicationsAccordion` and used there.